tools/decompress_baserom.py

```diff
@@ -32,31 +33,20 @@ def decompress(data: bytes, is_zlib_compressed: bool) -> bytes:
     return crunch64.yaz0.decompress(data)
 
 
-FILE_TABLE_OFFSET = {
-    "gc-eu-mq":         0x07170,
-    "gc-eu-mq-dbg":     0x12F70,
-}
-
-VERSIONS_MD5S = {
-    "gc-eu-mq":         "1a438f4235f8038856971c14a798122a",
-    "gc-eu-mq-dbg":     "f0b7f35375f9cc8ca1b2d59d78e35405",
-}
-
-
 def round_up(n, shift):
     mod = 1 << shift
     return (n + mod - 1) >> shift << shift
 
 
-def as_word_list(b) -> list[int]:
+def as_word_list(b) -> List[int]:
```

Comment:
Why use `List` when we already can use plain `list`?

Reply:
Compatibility with python 3.8

Reply:
That's why we have the `from __future__ import annotations` at the top of this file

Reply:
Oh, good to know. Maybe I'll revert all the other typing changes we made to support fig's ancient python version...